scripts/generate-cities-cache.js

```diff
@@ -1,76 +1,33 @@
+// @ts-check
+
 /**
  * Run this with: $ yarn generate-cities-cache
  */
-
-const fs = require("fs")
-const https = require("https")
-const path = require("path")
 const queryMap = require("./queryMap")
+const preheatGraphQLCache = require("./preheatGraphQLCache")
+const moment = require("moment")
+
+// @ts-ignore
 const cities = require("../data/cityDataSortedByDisplayPreference.json")
 
 const QUERY_NAME = "MapRendererQuery"
-const TTL = 3600 // 1 hour
-
 const MAX_GRAPHQL_INT = 2147483647
+const TTL = 3600 // 1 hour
+const FRESHNESS = moment()
+  .add(1, "month")
+  .toDate()
```

Comment:
For City data we set the freshness to be 1 month [from the time we cut the release].